src/nxt_cgroup.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (C) Andrew Clayton
+ * Copyright (C) F5, Inc.
+ */
+
+#include <nxt_main.h>
+
+#include <nxt_cgroup.h>
+
+
+static int nxt_mk_cgpath_relative(nxt_task_t *task, const char *dir,
+    char *cgpath);
+static nxt_int_t nxt_mk_cgpath(nxt_task_t *task, const char *dir,
+    char *cgpath);
+
+
+nxt_int_t
+nxt_cgroup_proc_add(nxt_task_t *task, nxt_process_t *process)
+{
+    int        len;
+    char       cgprocs[NXT_MAX_PATH_LEN];
```

Comment:
Please use MAX_PATH as discussed the other day.  It is standard (POSIX), and also more commonly used in Unit.

Reply:
Like I originally had it!

Reply:
Yep :)

BTW, I had a typo here.  I meant `PATH_MAX`.